### DISCUSSION TITLE
Input sequence days used

### DISCUSSION THREAD

**Discussion by ybl2333** [PaddlePaddle/PaddleSpatial#21] [BaiduKDDCup2022] [answered]
Dear organizers,

Are there any restrictions on the length of past data series used in the prediction of the next two days,
and will there be a public score list

Thank you for your time.

**Comment by zhoujingbo** [comment]
Hi, 

What do you mean of "a public score list" ?  

Thanks a lot.
Regards
Jingbo

**Comment by zhoujingbo** [comment, marked as answer]
> Dear organizers,
> 
> Are there any restrictions on the length of past data series used in the prediction of the next two days,

Hi, we set the maximum length of the input time-series 𝐿𝑥 as 14 days.


 and will there be a public score list
> 
> Thank you for your time.
